services/dask-sidecar/src/simcore_service_dask_sidecar/tasks.py

```diff
@@ -64,8 +59,12 @@ async def dask_setup(worker: distributed.Worker) -> None:
     # set up logging
     logging.basicConfig(level=settings.LOG_LEVEL.value)
     logging.root.setLevel(level=settings.LOG_LEVEL.value)
-    config_all_loggers(settings.DASK_LOG_FORMAT_LOCAL_DEV_ENABLED)
     logger.setLevel(level=settings.LOG_LEVEL.value)
+    # NOTE: Dask attaches a StreamHandler to the logger in distributed
+    # stopping the propagation to prevent double logging
+    for handler in logging.getLogger("distributed").handlers:
+        logging.getLogger("distributed").removeHandler(handler)
+    config_all_loggers(settings.DASK_LOG_FORMAT_LOCAL_DEV_ENABLED)
```

Comment:
Thanks for this one, let me know if you think it will be better to move it globally to the config_all_loggers() function. (potentially to remove all Handlers from all loggers, and leave only the root logger with one stream handler)